### ISSUE TITLE
ERROR:check_pkl_issuedate, IssueDate is post dated.  Is this error right?

### ISSUE THREAD

**Issue by jamiegau** [Ymagis/ClairMeta#158]
Hi,
Just reading this error..
ERROR:check_pkl_issuedate
PKL_9572db6b-6f32-40fc-806d-6a6e6398c0e8.xml : IssueDate is post dated : 2020-08-13 12:15:33
ERROR:check_cpl_issuedate
CPL_f7e1efdf-a008-41eb-b25d-ebb4743f4e40.xml : IssueDate is post dated : 2020-08-13 12:15:33

This indicates to me that he CPL date found in the CPL name
CorowaDistilCo_ADV-25_F-185_EN_51_2K_NULL_20200813_NUL_SMPTE_OV
in this case...  20200813 is after the date 2020-08-13 12:15:33 found in the PKL and CPL.  as 20200813 in the CPL should be counted as 
20200813 23:59:59.  it looks like its be compared as 20200813 00:00am, making the pkl/cpl date AFTER that date.  false positive being the result.

That's not right is it..  

Am reading this right?

Thanks..
James

**Comment by jamiegau** [comment]
Ok, I have been looking deeper into why this issue happens.
It looks like its connected to how I am using it in a docker container directly from a python application.
The problem seems to occur if asdcp is not found.  This is what I saw when I was on the production server reproducing the problem (Outside of the docker container).  Once I installed asdcplib, the error was no longer apparent.

The docker container does have asdcplib installed..  but I am calling clairmeta directly from my app with clairmeta being a subproject from my fork.

But initial look, /usr/local/bin is in the default path in docker containers.  It is when I "docker-compose exec container-name sh"
the asdcplib tools work fine.

So at this stage it looks like I need to track down how the asdcp tools are dealt with when clairmeta is directly called from python code..
Does this have an effect on how it works and the use of asdcplib executables..

Has anyone used it like this or seen similar problems?

Comments welcome.

**Comment by jamiegau** [comment]
Ok,
This issues turned out to be connected with the timezone set on my mediaconvert server.  It was UTC, and not set to local timezone (Australia/Melbourne).  I fixed this,
But it could also be connected with the time difference between when I created the DCP and how I then used clairmeta on it within the time zone difference..  If I waited longer, the problem may not have show also.

This is such a edge case, I'm an no longer going to look into it.
But it is now documented for those who may hit this issue..
